apps/website/src/examples/Breadcrumbs.customOverflowBackButton.tsx

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Breadcrumbs, Button, Tooltip, IconButton } from '@itwin/itwinui-react';
+import { SvgMore } from '@itwin/itwinui-icons-react';
+
+export default () => {
+  const items = Array(10)
+    .fill(null)
+    .map((_, index) => <Button key={index}>Item {index}</Button>);
+
+  return (
+    <div style={{ maxWidth: '50%', border: '1px solid lightpink', padding: 8 }}>
+      <Breadcrumbs
+        overflowButton={(visibleCount: number) => {
+          const previousBreadcrumb =
+            visibleCount > 1 ? items.length - visibleCount : items.length - 2;
+          return (
+            <Tooltip content={`Item ${previousBreadcrumb}`} placement='bottom'>
+              <IconButton
+                onClick={() => {
+                  // open previous breadcrumb
```

Comment:
Did you forget to push some code? This example is not very helpful if nothing happens on click.

Reply:
So this example is supposed to be a 'back' button to the last hidden breadcrumb. Nothing happens when I click a regular breadcrumb, so why should something happen when this button is clicked? Should something be happening when breadcrumbs are clicked? None of our examples are interactive like that.

Reply:
Because you are specifically trying to show a case where something different should happen for this overflowButton (otherwise why even show the example?)

There is always a fine line between overdocumenting and not documenting enough. So you could enhance this example to maintain the "current" state which changes when you click on the different buttons. But I think in this case it might be enough to not even show an example.

Reply:
I removed it